test/object-store/c_api/c_api.c

```diff
@@ -233,8 +233,10 @@ int realm_c_api_tests(const char* file)
     CHECK_ERROR();
     assert(num_bars == 0);
 
-    assert(realm_refresh(realm));
+    bool did_refresh = false;
+    assert(realm_refresh(realm, &did_refresh))
```

Comment:
missing ; ...

Reply:
I am driving this blind since I don't have the setup to build core. Will try to fix

Reply:
fixed